### ISSUE TITLE
Add supported Node versions?

### ISSUE THREAD

**Issue by poshaughnessy** [charpeni/svelte-example#5]
Hi! Turned out on this computer I was on an old version of Node, v4.4.7. `npm run build` failed with:

```
> rollup -c

Unexpected token {
SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Module._extensions..js (module.js:416:10)
    at Object.require.extensions..js (/home/p.oshaughnes/dev/src/external/svelte-example/node_modules/rollup/bin/rollup:763:6)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/p.oshaughnes/dev/src/external/svelte-example/node_modules/rollup-plugin-svelte/dist/rollup-plugin-svelte.cjs.js:4:14)
    at Module._compile (module.js:409:26)
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki

npm ERR! Linux 4.4.0-53-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
npm ERR! node v4.4.7
npm ERR! npm  v2.15.8
npm ERR! code ELIFECYCLE
npm ERR! svelte@0.1.0 build: `rollup -c`
npm ERR! Exit status 1
```

I've upgraded to shiny Node v7 now and it builds fine. Just wanted to share in case a) This helps anyone else, and b) you might like to add a quick mention of which versions of Node are supported to the README?

Thanks!

**Comment by chadananda** [comment]
I'm getting a similar error with updated node:

```
> rollup -c

module.js:472
    throw err;
    ^

Error: Cannot find module '../dist/rollup.js'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)

npm ERR! node v7.5.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! svelte@0.1.0 build: `rollup -c`
npm ERR! Exit status 1
```

**Comment by chadananda** [comment]
My issue was resolved with npm v4.2.0

**Comment by charpeni** [comment]
@poshaughnessy @chadananda Thank you. I'll add this to the README.
